### ISSUE TITLE
Hit a bug in the formatter when formatting lib/src/command_runner.dart

### ISSUE THREAD

**Issue by JakeChampion** [dart-lang/dart_style#859]
This came from a Travis CI build on the dart-lang/pub GitHub repository -- https://travis-ci.org/dart-lang/pub/jobs/590344960

```
Please report at github.com/dart-lang/dart_style/issues.
type '_PropertySelector' is not a subtype of type '_MethodSelector'
#0      new CallChainVisitor (package:dart_style/src/call_chain_visitor.dart:151:11)
#1      SourceVisitor.visitPropertyAccess (package:dart_style/src/source_visitor.dart:2109:5)
#2      PropertyAccessImpl.accept (package:analyzer/src/dart/ast/ast.dart:8492:49)
#3      SourceVisitor.visit (package:dart_style/src/source_visitor.dart:2416:10)
#4      SourceVisitor._visitAssignment (package:dart_style/src/source_visitor.dart:2519:5)
#5      SourceVisitor.visitAssignmentExpression (package:dart_style/src/source_visitor.dart:396:5)
#6      AssignmentExpressionImpl.accept (package:analyzer/src/dart/ast/ast.dart:747:49)
#7      SourceVisitor.visit (package:dart_style/src/source_visitor.dart:2416:10)
#8      SourceVisitor.visitExpressionStatement.<anonymous closure> (package:dart_style/src/source_visitor.dart:1139:7)
#9      SourceVisitor._simpleStatement (package:dart_style/src/source_visitor.dart:3194:9)
#10     SourceVisitor.visitExpressionStatement (package:dart_style/src/source_visitor.dart:1138:5)
#11     ExpressionStatementImpl.accept (package:analyzer/src/dart/ast/ast.dart:3768:49)
#12     SourceVisitor.visit (package:dart_style/src/source_visitor.dart:2416:10)
#13     SourceVisitor.visitBlock (package:dart_style/src/source_visitor.dart:496:7)
#14     BlockImpl.accept (package:analyzer/src/dart/ast/ast.dart:1111:49)
#15     SourceVisitor.visit (package:dart_style/src/source_visitor.dart:2416:10)
#16     SourceVisitor.visitTryStatement (package:dart_style/src/source_visitor.dart:2296:5)
#17     TryStatementImpl.accept (package:analyzer/src/dart/ast/ast.dart:9997:49)
#18     SourceVisitor.visit (package:dart_style/src/source_visitor.dart:2416:10)
#19     SourceVisitor.visitBlock (package:dart_style/src/source_visitor.dart:496:7)
#20     BlockImpl.accept (package:analyzer/src/dart/ast/ast.dart:1111:49)
#21     SourceVisitor.visit (package:dart_style/src/source_visitor.dart:2416:10)
#22     SourceVisitor.visitBlockFunctionBody (package:dart_style/src/source_visitor.dart:529:5)
#23     BlockFunctionBodyImpl.accept (package:analyzer/src/dart/ast/ast.dart:1066:49)
#24     SourceVisitor.visit (package:dart_style/src/source_visitor.dart:2416:10)
#25     SourceVisitor._visitBody (package:dart_style/src/source_visitor.dart:2664:5)
#26     SourceVisitor._visitMemberDeclaration (package:dart_style/src/source_visitor.dart:2613:5)
#27     SourceVisitor.visitMethodDeclaration (package:dart_style/src/source_visitor.dart:1924:5)
#28     MethodDeclarationImpl.accept (package:analyzer/src/dart/ast/ast.dart:7108:49)
#29     SourceVisitor.visit (package:dart_style/src/source_visitor.dart:2416:10)
#30     SourceVisitor._visitMembers (package:dart_style/src/source_visitor.dart:2561:7)
#31     SourceVisitor.visitClassDeclaration (package:dart_style/src/source_visitor.dart:717:5)
#32     ClassDeclarationImpl.accept (package:analyzer/src/dart/ast/ast.dart:1554:49)
#33     SourceVisitor.visit (package:dart_style/src/source_visitor.dart:2416:10)
#34     SourceVisitor.visitCompilationUnit (package:dart_style/src/source_visitor.dart:777:7)
#35     CompilationUnitImpl.accept (package:analyzer/src/dart/ast/ast.dart:2140:49)
#36     SourceVisitor.visit (package:dart_style/src/source_visitor.dart:2416:10)
#37     SourceVisitor.run (package:dart_style/src/source_visitor.dart:207:5)
#38     DartFormatter.formatSource (package:dart_style/src/dart_formatter.dart:153:26)
#39     processFile (package:dart_style/src/io.dart:77:28)
#40     processDirectory (package:dart_style/src/io.dart:58:10)
#41     formatPaths (file:///b/s/w/ir/cache/builder/sdk/third_party/pkg_tested/dart_style/bin/format.dart:253:12)
#42     main (file:///b/s/w/ir/cache/builder/sdk/third_party/pkg_tested/dart_style/bin/format.dart:189:5)
#43     _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:303:32)
#44     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:172:12)
```

**Comment by JakeChampion** [comment]
I'm not sure what else would be good for me to add to this issue, sorry 😔 

If there is anything else I should add, please let me know and I will add it :-)

**Comment by munificent** [comment]
Your report with the stack trace is fine. :)

I fixed this bug yesterday (#855), but it won't roll out until a new dev version of the Dart SDK ships. Sorry for the pain. :(
